tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/IMemento.java

```diff
@@ -1,13 +1,16 @@
-/*******************************************************************************
- * Copyright (c) 2003, 2007 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
+/**
```

Comment:
Hey Terry,  It isn't legal to change the copyright.  The original copyright holders still persist.  WE have to add a rule to SF to not perform the copyright check for these files.  

I don't think there is any point in source formatting these classes, since we may want to port bug fixes in the future for this class.